_posts/2018-05-17-introducing-watir-screenshot-stitch.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Introducing watir-screenshot-stitch
+date: 2018-05-17
+author: Sam Nissen
+author_url: https://about.me/samnissen
+comments: true
+categories: [Addons]
+---
+
+Selenium no longer supports full page screenshots,
+[(though it might again)](https://github.com/mozilla/geckodriver/issues/570).
+But that’s OK! Watir still can – there are many methods,
+and the [watir-screenshot-stitch](https://github.com/samnissen/watir-screenshot-stitch)
+gem employs two of them. It extends Watir::Screenshot to give you
+two new public methods:
+
+## Watir::Screenshot#save_stitch
+
+`save_stitch` will take images while paging down the page,
+and stitch them together with ImageMagick:
+
+```ruby
```

Comment:
This markdown doesn't work by default in jekyll. We need to use the liquid markdown:
{% highlight ruby %} gem install watir {% endhighlight %}